samples/scenarios/templates/db_impersonation/manifest.xml

```diff
@@ -4,7 +4,7 @@
   <vendor-information>
     <company name="Company"/>
     <support-link url = "https://github.com/tableau/connector-plugin-sdk"/>
-    <driver-download-link url = "some url"/>
```

Comment:
Have you double checked that this loads in Tableau? I know that we don't allow tableau domains and trademarks in SDK connectors for some fields, not sure if this is one of them. You can always direct link to the Impala driver if we flag this.

Reply:
I turned off signature verification when I ran the connector. It is probably because of that I did not get error with the link.

Reply:
I have replaced it with impala JDBC driver link/

Reply:
Sorry, missed this on the first iteration. I think its fine to link to this in the readme as instructions for the working example, but in the sample connector please just reuse the postgres_jdbc values for driver-download-link and support-link.